Assets/LeapMotionModules/ElementRenderer/Scripts/Editor/LeapGuiElementEditor.cs

```diff
@@ -0,0 +1,105 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEditor;
+using Leap.Unity;
+using Leap.Unity.Query;
+
+[CanEditMultipleObjects]
+[CustomEditor(typeof(LeapGuiElementData), editorForChildClasses: true, isFallback = true)]
+public class DefaultFeatureDataEditor : CustomEditorBase {
+  protected override void OnEnable() {
+    base.OnEnable();
+    dontShowScriptField();
```

Comment:
Hmm since it touches so many non-gui files, I think I'll change this in a different PR into develop instead of on this PR which is already pretty big.